src/Requests/GenerativeModel/GenerateContentRequest.php

```diff
@@ -54,5 +55,11 @@ protected function defaultBody(): array
             ),
             'generationConfig' => $this->generationConfig?->toArray(),
         ];
+
+        if ($this->systemInstruction !== null) {
```

Comment:
If it is not a problem to send null for this parameter, it can be written with the null-safe operator instead of the condition.